CMakeLists.txt

```diff
@@ -605,6 +605,10 @@ option(SWIFT_ENABLE_EXPERIMENTAL_CONCURRENCY
   "Enable build of the Swift concurrency module"
   FALSE)
 
+option(SWIFT_ENABLE_EXPERIMENTAL_CXX_INTEROP
```

Comment:
It is not clear to me why we would need a second flag to enable/disable building C++ interop in addition to [`SWIFT_BUILD_STDLIB_CXX_MODULE`](https://github.com/apple/swift/blob/8202acb4ea23b84acbe10ec7eec02555f6066a85/CMakeLists.txt#L1273) -- from the logic in the [main CMakeLists.txt](https://github.com/apple/swift/blob/8202acb4ea23b84acbe10ec7eec02555f6066a85/CMakeLists.txt#L1263), either C++ interop is built as part of the `stdlib` tree, or it is "pulled in" if we don't want to build the stdlib, so a single flag seems enough to control this (but it could be I'm missing how Windows builds)

Reply:
Yeah, I agree with you on the single flag.  I think that the general pattern the other modules include is that you cannot build them without the standard library build.  This flag is matching the existing pattern and paths.  The extra flag is for the c++ special behavior of building that without the standard library.

Reply:
That's a fair point -- I agree it's best to align with the current pattern.